notebooks/confirm_gpu_available.ipynb

```diff
@@ -0,0 +1,218 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "16e3552b-6245-4e8a-ab1c-2e17729901d4",
+   "metadata": {},
+   "source": [
+    "# Confirm Notebook Can Access GPU \n",
+    "\n",
+    "This notebook provides a couple of small examples that confirm your notebook environment is able to connect to a GPU, either locally or on a cluster. The bottom section also provides a short example showcasing the increased performance of using the GPU over a CPU for a matrix multiplication task. \n",
+    "\n",
+    "\n",
+    "The one caveat to ensuring this notebook works correctly and you can communicate with the gpu, is that you must install pytorch.\n",
+    "\n",
+    "```\n",
+    "pip3 install torch==1.9.0+cu114 torchvision==0.10.0+cu114 torchaudio==0.9.0 -f https://download.pytorch.org/whl/torch_stable.html\n",
```

Comment:
```suggestion
    "pip3 install torch==1.9.0+cu111 torchvision==0.10.0+cu111 torchaudio==0.9.0 -f https://download.pytorch.org/whl/torch_stable.html\n",
```
Hmm I only see cuda 11.1 wheels on https://download.pytorch.org/whl/torch_stable.html. Running this pip install gives me an error:

![Screenshot from 2021-09-22 10-57-59](https://user-images.githubusercontent.com/15005500/134370252-83bb172f-a17c-45db-a006-74e02e94cd31.png)

Reply:
great catch @chauhankaranraj that is absolutely the wrong request. :)